CONTRIBUTING.md

```diff
@@ -30,7 +30,7 @@ Pull requests containing contributions of any significant changes should be acco
 
 ### Customizations and Tests
 
-Customizations to the schema should be placed in the `customizations` folder, named as `mei-NAME-[suffix].xml`. Our automated build system, TravisCI, will build all customizations in this folder and ensure there are no problems. It will also run our automated tests to ensure no validation problems.
+Customizations to the schema should be placed in the `customizations` folder, and named according to the scheme: `mei-NAME-`[suffix].xml`. Our automated build system, TravisCI, will build all customizations in this folder and ensure there are no problems. It will also run our automated tests to ensure no validation problems.
```

Comment:
```suggestion
Customizations to the schema should be placed in the `customizations` folder, and named according to the scheme: `mei-NAME-`[suffix].xml`. Our automated build system will build all customizations in this folder and ensure there are no problems. It will also run our automated tests to ensure no validation problems.
```

Reply:
@musicEnfanthen I fixed the reference to TravisCI here.

Reply:
Thanks @ahankinson !